### PR TITLE
New Post: Optimizing your Server

### PR DESCRIPTION
**Pull request by AlbertPM9000**

### CODE REVIEW

File: docs/minecraft/general/optimizing-your-server.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title:  "Optimzing your Server"
```
Comment (itsjuoum) [inline docs/minecraft/general/optimizing-your-server.md:3]: Correct the spelling from "Optimzing" to "Optimizing"

**Review by itsjuoum — CHANGES_REQUESTED** [review]

File: docs/minecraft/general/optimizing-your-server.md (hunk 1 above)
Comment (AlbertPM9000) [reply docs/minecraft/general/optimizing-your-server.md]: Done. Can you see it or do I have to open a new request?

**Comment by KorbsStudio** [comment]
Please also mention other information about making sure customers are using the proper version of Java for their selected version of Minecraft and also making sure that they have enough resources used for their server like the correct amount of CPU cores and memory as they can have an impact on the server if not setup properly. 

or...I can do this myself, but I'm just giving you a chance.

**Comment by KorbsStudio** [comment]
Duplicate: #30

**Pull request closed without merging**